service/service_helper.h

```diff
@@ -0,0 +1,76 @@
+#ifndef SERVICE_HELPER_H_
+#define SERVICE_HELPER_H_
+
+#include "service.pb.h"
+#include "unique_id.h"
+#include <atomic>
+#include <chrono>
```

Comment:
<chrono> is an unapproved C++11 header. 
 <mutex> is an unapproved C++11 header.?